src/Illuminate/Support/Str.php

```diff
@@ -736,31 +736,34 @@ public static function password($length = 32, $letters = true, $numbers = true,
         $characters = [];
 
         if ($letters) {
-            $characters = array_merge($characters, [
-                'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k',
-                'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v',
-                'w', 'x', 'y', 'z', 'A', 'B', 'C', 'D', 'E', 'F', 'G',
-                'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R',
-                'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z',
-            ]);
+            $characters = [
+                ...$characters,
+                ...range('a', 'z'),
+                ...range('A', 'Z'),
+            ];
         }
 
         if ($numbers) {
-            $characters = array_merge($characters, [
+            $characters = [
+                ...$characters,
                 '0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
```

Comment:
I know this is closed, but I'm really curious why you didn't use `range(0, 9)` here?

Reply:
that generates `int`s, not `string`s.

probably would've worked okay with PHP's type inference on concatenation, but Taylor specifically made these string integers, and I couldn't replicate that with `range()`.